doc/api/os.md

```diff
@@ -368,6 +368,15 @@ changes:
 
 Returns the system uptime in number of seconds.
 
+Please be aware that the value returned can be inaccurate in some
+rare virtualization cases. The issue arises when the virtualized
+guest instance shares the kernel with the host system due to a bug
+in libuv. `os.uptime()` may thus provide the host's uptime instead of guest's.
```

Comment:
The way this sentence is worded it implies that the bug in libuv is what is responsible for the guest instance sharing the kernel with the host system.